worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/AreaPickaxe.java

```diff
@@ -76,6 +78,9 @@ public boolean actPrimary(Platform server, LocalConfiguration config, Player pla
 
                             editSession.setBlock(x, y, z, BlockTypes.AIR.getDefaultState());
 
+                            BlockVector3 pos = BlockVector3.at(x, y, z);
+                            ((World) clicked.getExtent()).queueBlockBreakEffect(server, pos, initialType,
+                                    clicked.toVector().toBlockPoint().distanceSq(pos));
```

Comment:
This should probably only be done if the setBlock method returns true

Reply:
It should be noted upstream doesn't check for this, but sure.

Reply:
In that case also ensure to wrap in //FAWE start and end comments